delete_emoji.py

```diff
@@ -16,5 +16,8 @@
     emoji_to_delete = emoji
 
 for name, url in emoji_to_delete.items():
-    client.remove_emoji(name)
-    print("deleted {}".format(name))
+    try:
+        client.remove_emoji(name)
+        print("deleted {}".format(name))
+    except:
```

Comment:
Does this actually raise?  Thought it returned an error response

Reply:
The request returns an error response, but I guess the slack SDK raises

Reply:
Did you confirm this worked though?  If it's just an error response how would it ever catch?

Reply:
I did confirm. It would be just an error response if we were just straight `curl`ing or something, but the SDK wraps it into a raised error